CombinatorialGames/Game/Small.lean

```diff
@@ -0,0 +1,61 @@
+/-
+Copyright (c) 2025 Tristan Figueroa-Reid. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Tristan Figueroa-Reid
+-/
+import CombinatorialGames.Surreal.Basic
+
+/-!
+# Small games all around
+
+We define small games, games that are smaller than every Surreal
+and dicotic games, games `G` where both players can move from every nonempty subposition of `G`.
+
+## TODO
+
+- Define infinitesimal games as games x such that ∀ y : ℝ, 0 < y → -y < x ∧ x < y
```

Comment:
It might be worth first adopting a simpler definition: a game `x` is infinitesimal if for every positive short numeric game `y`, `-y < x ∧ x < y`. We can prove this equivalence later.

Reply:
A short game `x` is infinitesimal if it meets those criteria - not a long game.